### DISCUSSION TITLE
기존 방식이 오늘자로 먹히지 않습니다.

### DISCUSSION THREAD

**Discussion by jhdaniel99** [changdoc/naver-vaccine-macro#121] [Q&A]
20일자로 바뀐 방식( 2000ms, 한 창에서만)으로 진행했으나 잘못된 접근이라며 시행을 차단당했습니다.
딜레이의 문제인지, 해당 방식이 아예 무효화된 건지는 확인해보아야 알 것 같습니다.

**Comment by changdoc** [comment]
음.. 전 수분 동안 차단이 안되는데
https://github.com/changdoc/naver-vaccine-macro/discussions/51
여기 내용 처럼 개발자 도구에서 2000 미만으로 새로고침 되고 있는건 아닌지 확인 해보세요.

**Reply by Denhze** [reply]
저도 오늘 잘 되고 있습니다.
크롬 1, 웨일 1, 두개 계정으로 2000 으로 진행중 입니다.

**Reply by Jayyyy-04** [reply]
제 지인 10분 전 성공하였습니다. 작성자분 개인의 문제로 보입니다.

**Reply by jin0906** [reply]
저도 3시간전에 성공해서 백신 맞고 왔습니다. 작성자분 개인의 문제 같습니다
